tests/methods/ParseTest.php

```diff
@@ -137,6 +137,12 @@ public function test_Piwik_data() {
      * @depends testSepRowAutoDetection
      */
     public function testGetColumnDatatypes() {
+        if (!function_exists('array_column')) {
```

Comment:
Oh, okay. But for security and performance reasons everbody should use a higher version. ☺️